### DISCUSSION TITLE
Move focus to the next window in current mode

### DISCUSSION THREAD

**Discussion by mingsu** [rxhanson/Rectangle#369] [Ideas]
It will be great if this app can integrate the functionality of switch focuses between windows in current mode.

For example,

in the current space, three windows were arranged in: top-left, bottom-left, and half-right.

Many times, we have to switch the focus temporary to the bottom-left window from half-right window, it would be great accomplished by a short cut, such as meta-j (for clock-wise loop) and meta-k (for reverse clock-wise loop)

**Comment by rxhanson** [comment]
This request has come up before, and I do like the idea. It should be possible to add this type of behavior in, I just don't have the bandwidth to add it in at the moment. I would definitely consider a pull request for it.

**Reply by mingsu** [reply]
That will be excellent!

**Comment by timtyrrell** [comment]
I am seeking a Mac solution for this very problem right now. Well, more of a "focus on window to the right/left/up/down of the current focused window"
